README.md

```diff
@@ -1,13 +1,184 @@
-# Kin core Stellar SDK for Android
+# Kin core SDK for Android
 Android library responsible for creating a new Stellar account and managing KIN balance and transactions.
 ![Kin Token](kin_android.png)
 
-## Disclaimer
+## Build
+
+* Add this to your module's `build.gradle` file.
+```gradle
+repositories {
+    ...
+    maven {
+        url 'https://jitpack.io'
+    }
+}
+...
+dependencies {
+    ...
+
+    compile "com.github.kinfoundation:kin-core-android:<latest version>"
+}
+```
+
+## Usage
+### Connecting to a service provider
+Create a new `KinClient` with two arguments: an android `Context` and a `ServiceProvider`. 
+
+A `ServiceProvider` provides details of how to access the Stellar horizon end point.
+The example below creates a `ServiceProvider` that will be used to connect to the main (production) Stellar 
+network
+```java
+ServiceProvider infuraProvider =  
+    new ServiceProvider("https://horizon.stellar.org", ServiceProvider.NETWORK_ID_MAIN));
+KinClient kinClient = new KinClient(context, infuraProvider);
+```
+
+To connect to a test Stellar network use the following ServiceProvider:
+```java
+new ServiceProvider("https://horizon-testnet.stellar.org", ServiceProvider.NETWORK_ID_TEST)
+``` 
+
+### Creating and retrieving a KIN account
+The first time you use `KinClient` you need to create a new account, using a passphrase. 
```

Comment:
It is misleading because you really create a Keypair and not an account.

Reply:
I changed it to `addAccount` as `createAccount` is misleading because it doesn't exist anymore.
anyhow, I don't want to expose the notion of keypair vs account on  Stellar blockchain, I did detail in the onborading section, that account needs to be created on the blockchain by a different entity before it can be used.